tox.ini

```diff
@@ -12,8 +12,11 @@ minversion = 2.9
 description = "Scapy unit tests"
 whitelist_externals = sudo
 setenv = SCAPY_ROOT_DIR={env:PWD}
```

Comment:
Small question not related to this PR: what is this used for ? I couldn't find any other occurence of SCAPY_ROOT_DIR
Moreover, PWD doesn't exist on windows, making tox crash-on-start

Reply:
You can try to use nested env substitutions like `{env:KEY:{env:DEFAULT_OF_KEY}}`. That should allow to support Windows easily.